### PR TITLE
Watch subcomponents

### PR DESCRIPTION
**Pull request by zkayyali812**
Signed-off-by: Zachary Kayyali <zkayyali@redhat.com>
https://github.com/open-cluster-management/backlog/issues/15633

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -99,13 +106,13 @@ func main() {
...
-		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "797f9276.open-cluster-management.io",
-		// LeaderElectionNamespace: "backplane-operator-system", // Ensure this is commented out. Uncomment only for running operator locally.
+		Scheme:                  scheme,
+		MetricsBindAddress:      metricsAddr,
+		Port:                    9443,
+		HealthProbeBindAddress:  probeAddr,
+		LeaderElection:          enableLeaderElection,
+		LeaderElectionID:        "797f9276.open-cluster-management.io",
+		LeaderElectionNamespace: "backplane-operator-system", // Ensure this is commented out. Uncomment only for running operator locally.
```
Comment (JakobGray) [inline main.go:115]: Is this comment no longer true? And if the namespace is hardcoded to that then I think we'll have the same problem we had in https://github.com/open-cluster-management/backlog/issues/15203

Comment (zkayyali812) [reply main.go:115]: Yes... Good catch! I uncomment this when running it locally using `make run`, and forgot to comment it back. Will fix.

**Review by JakobGray — APPROVED** [review]
It looks like the fix for the watch concerns here is to install the CRDs before running the controller. The only concern I see with that is we won't reconcile the CRDs if the user deletes one of them, which I'm fine with.

**Pull request merged**